packages/manager/modules/billing/src/payment/method/add/routing.js

```diff
@@ -125,7 +126,12 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
         return goPaymentList(
           {
             type: 'success',
-            text: $translate.instant('billing_payment_method_add_success'),
+            text: $translate.instant(
+              selectedPaymentMethodType.paymentType ===
+                OVH_PAYMENT_METHOD_TYPE.BANK_ACCOUNT
+                ? 'billing_payment_method_add__sepa_success'
```

Comment:
'billing_payment_method_add__sepa_success' => 'billing_payment_method_add_sepa_success' 
We can delete one _